packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -799,6 +799,32 @@ class PouchLink extends CozyLink {
     }
     this.pouches.syncImmediately()
   }
+
+  /**
+   * Adds a new doctype to the list of managed doctypes, sets its replication options,
+   * adds it to the pouches, and starts replication.
+   *
+   * @param {string} doctype - The name of the doctype to add.
+   * @param {Object} options - The replication options for the doctype.
+   */
+  addDoctype(doctype, options) {
+    this.doctypes.push(doctype)
+    this.doctypesReplicationOptions[doctype] = options
+    this.pouches.addDoctype(doctype, options)
+    this.startReplication()
```

Comment:
The replication start is arguable: I'm not sure this is something we should expect from a `addDoctype` method

Reply:
But I don't we where else the replication will/should be started. Should it be started by the caller of addDoctype ?

Reply:
Ok. Let's give this responsibility to the caller,  then

Reply:
Done